riff-raff/app/deployment/preview/Preview.scala

```diff
@@ -0,0 +1,45 @@
+package deployment.preview
+
+import cats.data.Validated
+import cats.data.Validated.{Invalid, Valid}
+import cats.kernel.Semigroup
+import cats.syntax.semigroup._
+import com.gu.management.Loggable
+import magenta.`package`._
+import magenta.artifact.S3YamlArtifact
+import magenta.deployment_type.DeploymentType
+import magenta.graph.{DeploymentTasks, Graph}
+import magenta.input.resolver._
+import magenta.input.{ConfigErrors, Deployment}
+import magenta.{DeployParameters, DeploymentResources}
+
+object Preview extends Loggable {
```

Comment:
Yes - I have come to that conclusion whilst working on the filtering functionality! I've renamed the controller to `PreviewController` to make it obvious and have prefixed `Legacy` on the front of all the JSON preview classes.